ui/app/AppLayouts/Chat/panels/UserListPanel.qml

```diff
@@ -79,10 +79,11 @@ Item {
             section.property: "onlineStatus"
             section.delegate: (root.width > 58) ? sectionDelegateComponent : null
             delegate: StatusMemberListItem {
+                readonly property bool ensVerified: Utils.isEnsVerified(model.pubKey)
                 width: ListView.view.width
                 nickName: model.localNickname
-                userName: !!model.ensName ? "@" + Utils.removeStatusEns(model.ensName) : model.displayName !== "" ? model.displayName : model.alias
-                pubKey: !!model.ensName ? "" : Utils.getCompressedPk(model.pubKey)
+                userName: ensVerified ? model.ensName : model.displayName !== "" ? model.displayName : model.alias
```

Comment:
It's still unclear why we need resolve between `ensName`, `displayName` and `alias` for displaying, but probably it's a subject for a separate issue :)

Reply:
Yeah it's a behavior/design decision; not sure if the component itself should decide. So if there's an "ensName" then we display that but also don't display pubkey or id ring